.expeditor/buildkite/verify.ps1

```diff
@@ -5,7 +5,7 @@ ruby -v
 bundle --version
 
 echo "--- bundle install"
-bundle config set --local without tools maintenance deploy
+bundle config set --local without deploy
```

Comment:
I think we want to exclude kitchen here too. Why would we want to exclude it on the unix-based tests but not on the Windows based tests?

Reply:
Yes, good question. I choose not to include it here because we don't have the ruby 3.0 test running for Windows in we verify pipeline.